lib/web/apiserver_test.go

```diff
@@ -1920,7 +1920,6 @@ func TestWebAgentForward(t *testing.T) {
 }
 
 func TestActiveSessions(t *testing.T) {
-	t.Parallel()
 	s := newWebSuite(t)
```

Comment:
While you're here, I would also move `SetTestModules` to the very first line of the test.

Probably not a good idea to start a test auth server in OSS mode and then after it's already running tell it that it's now enterprise.